src/PacketHeader.h

```diff
@@ -129,7 +129,7 @@ class PacketHeader : public QObject
     /// \brief Parse the packet header and take appropriate measures (like change settings, or
     /// quit the program if peer settings don't match)
     virtual void parseHeader()                          = 0;
-    virtual void checkPeerSettings(int8_t* full_packet) = 0;
+    virtual bool checkPeerSettings(int8_t* full_packet) = 0;
```

Comment:
Will add the comment. Not sure we need the [[nodiscard]] though. The return value is just a convenience I added - checkPeerSettings sends out the signal that ultimately shuts down the jacktrip object if the peer settings don't match. I just use the return value to avoid doing any further network setup since it's just going to be torn down.